backstage-helm/values.yaml

```diff
@@ -140,6 +118,8 @@ upstream:
       value: changeme
     - name: CONFLUENCE_PASSWORD
       value: changeme
+    - name: LOG_LEVEL
```

Comment:
I'd remove this as it was just for debug purposes

Reply:
done